rai/node/bootstrap.cpp

```diff
@@ -5,9 +5,14 @@
 
 #include <boost/log/trivial.hpp>
 
-// Updated 1-27-18
+constexpr double bootstrap_connection_scale_target = 50000.0;
```

Comment:
Can you change these to specify the units in the identifier?

e.g. `bootstrap_minimum_block_rate_blocks_per_sec` or `bootstrap_minimum_block_rate_bps`, `bootstrap_minimum_termination_time_sec`, etc.